umbraco/Umbraco-CMS
# V14: Don't add published culture infos if not published

# Notes
Problem right now is that when we `Unpublish` a single culture, that re-published the content, but just without the "unpublished" culture.

When we unpublish all cultures, it works a bit different though. We don't unpublish the cultures. We just unpublished the content itself, and thus the cultures are not routable.
Problem is though the `content.PublishedInfos` still containted the "Published" cultures.

This PR remedies that by now publishing all cultures instead.

# How to test
- Have some languages on your site
- Make a document type and allow as root & allow vary by culture
- Add a Title property to the document
- Make some content in the different languages
- Ensure you can publish & unpublish a single culture
- Ensure you can publish & unpublish ALL cultures
- Ensure rollback works.
